DarkDream99/AlgoBridge-web
# Configure webpack

News:

- Add eslint
- Add modules analyzer
- Add .env config to manipulate with environment variables

HOW TO TEST:

1. Pull this branch.
2. Run: `npm install`.
3. Run: `npm run server` to execute server and then go to the url (generated by script) and check that app works properly.
4. To connect correctly to the server API config correctly .env file in the root directory of the project.
5. Check building and dev verstions via commands: `npm run dev`, `npm run prod`.
6. Check modules analyzer via command: `npm run analyzer`
